test/migration/verify/verify_migration_test.go

```diff
@@ -119,6 +122,44 @@ func verifySecondMemberProvisionedSpace(t *testing.T, awaitilities wait.Awaitili
 	cleanup.AddCleanTasks(t, awaitilities.Host().Client, userSignupForSpace)
 }
 
+func verifyProvisionedSubSpace(t *testing.T, awaitilities wait.Awaitilities) {
+	parentSpace, _ := VerifyResourcesProvisionedForSpace(t, awaitilities, migration.ProvisionedParentSpace)
+	userSignupForSpace := checkMURMigratedAndGetSignup(t, awaitilities.Host(), migration.ProvisionedParentSpace)
+	targetClusterRoles := []string{cluster.RoleLabel(cluster.Tenant)}
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member2()
+	memberCluster, found, err := hostAwait.GetToolchainCluster(t, cluster.Member, memberAwait.Namespace, nil)
+	require.NoError(t, err)
+	require.True(t, found)
+	subSpaceNamespace := GetDefaultNamespace(parentSpace.Status.ProvisionedNamespaces)
+
+	subSpace, err := awaitilities.Host().WaitForSubSpace(t,
+		migration.ProvisionedSpaceRequest,
+		subSpaceNamespace,
+		parentSpace.GetName(),
+		wait.UntilSpaceHasTargetClusterRoles(targetClusterRoles),
+		wait.UntilSpaceHasTier("base"),
+		wait.UntilSpaceHasAnyProvisionedNamespaces())
+	require.NoError(t, err)
+
+	spaceRequest, err := memberAwait.WaitForSpaceRequest(t,
+		types.NamespacedName{
+			Namespace: subSpaceNamespace,
+			Name:      migration.ProvisionedSpaceRequest,
+		},
+		wait.UntilSpaceRequestHasStatusTargetClusterURL(memberCluster.Spec.APIEndpoint),
+		wait.UntilSpaceRequestHasNamespaceAccess(subSpace),
+		wait.UntilSpaceRequestHasConditions(wait.Provisioned()),
+	)
+	require.NoError(t, err)
+	VerifyNamespaceAccessForSpaceRequest(t, memberAwait.Client, spaceRequest)
+
+	cleanup.AddCleanTasks(t, awaitilities.Host().Client, subSpace)
+	cleanup.AddCleanTasks(t, awaitilities.Member2().Client, spaceRequest)
+	cleanup.AddCleanTasks(t, awaitilities.Host().Client, parentSpace)
```

Comment:
the subSpace is managed by SpaceRequest, if you try to delete it before deleting SpaceRequest, then the SpaceRequest controller will re-create it.
technically speaking, the only thing you need to do is to delete the parent-space, the rest will be cleaned up automatically by the GC and via finalizers
```suggestion
	cleanup.AddCleanTasks(t, awaitilities.Host().Client, parentSpace)
```

Reply:
Fixed in 30ff3cf.